src/Story/Preview/PreviewDataParser.php

```diff
@@ -0,0 +1,37 @@
+<?php declare(strict_types=1);
+
+namespace Torr\Storyblok\Story\Preview;
+
+use Torr\Storyblok\Exception\Story\InvalidDataException;
+
+final class PreviewDataParser
+{
+	public static function parse (?string $editableData) : ?array
```

Comment:
You should probably allow `mixed` here, as the type is never validate elsewhere and this might lead to an unexpected type error.

Reply:
And possibly throw an exception if it's not `string` or `null`.